app/models/account.rb

```diff
@@ -26,6 +26,7 @@ def price_groups
   accepts_nested_attributes_for :account_users
 
   scope :active, lambda {{ :conditions => ['expires_at > ? AND suspended_at IS NULL', Time.zone.now] }}
+  scope :global_account_types, -> { where("accounts.type in (:global)", { global: config.global_account_types }) }
```

Comment:
Curly braces around `global: config.global_account_types` aren't necessary. Matter of personal preference, but I tend to only use named parameters (e.g. `:global` when I have multiple variables or I want to use a single value in multiple places. `where("accounts.type in ?", config.global_account_types)` is a little more succinct.

In this case I might also use `where(accounts: { type: config.global_account_types })`.

Not a strong feeling on it, though.